src/Rule/AtLeast.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Validator\Rule;
+
+use Yiisoft\Validator\Rule;
+use Yiisoft\Validator\Result;
+use Yiisoft\Validator\DataSetInterface;
+
+/**
+ * AtLeastValidator checks if at least $min of many attributes are filled.
+ */
+class AtLeast extends Rule
+{
+    /**
+     * The minimum required quantity of filled attributes to pass the validation.
+     * Defaults to 1.
+     */
+    private int $min = 1;
+
+    /**
+     * The list of required attributes that will be checked.
+     */
+    private array $attributes = [];
+
+    /**
+     * The list of alternative required attributes that will be checked.
+     */
+    private array $alternativeAttributes = [];
```

Comment:
I don't think it's needed. If we need multiple groups we can add multiple validators, can't we?

Reply:
I updated the PR. I have a question related to errrors messages by attribute this line: https://github.com/developeruz/yii2/blob/083da138572d34aae9212d4249d0c64e2cdc2277/framework/validators/AtLeastValidator.php#L102 

I saw that in yii-3 is different. Is necessary this feature?.